### PR TITLE
chore: adjust positioning and spacing of amount hint

### PR DESCRIPTION
**Pull request by dated**
## Summary

Adjustments to the spacing of the amount component, mainly for when the hint is displayed.

## Checklist

-   [ ] My changes look good in both light AND dark mode
-   [ ] The change is not hardcoded to a single network, but has multi-asset in mind
-   [ ] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/domains/transaction/components/TransactionDetail/TransactionAmount/TransactionAmount.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const TransactionAmount: React.FC<TransactionAmountProperties> = ({
 			{...properties}
 		>
 			<Label color={isSent ? "danger" : "success"}>
-				<div className="flex space-x-0.5 -mx-1">
+				<div className="flex space-x-2">
```
Comment (w3ea) [inline src/domains/transaction/components/TransactionDetail/TransactionAmount/TransactionAmount.tsx:80]: what's the reason for these changes? :hmhm:
because it doesn't match the design


image (2)

Screenshot 2021-09-02 133014

File: src/app/components/Label/Label.styles.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import tw from "twin.macro";
 import { Size } from "types";
 
-const baseStyle = tw`inline-block font-semibold border-2 rounded`;
+const baseStyle = tw`inline-block font-semibold border-2 rounded overflow-hidden`;
```
Comment (w3ea) [inline src/app/components/Label/Label.styles.tsx:4]: and this change create alignment issues

image (1)

https://user-images.githubusercontent.com/24996341/131825637-a724f6e0-1728-4c74-95aa-54041853b042.mp4

Comment (dated) [reply src/app/components/Label/Label.styles.tsx:4]: good catch :hmhm:

File: src/app/components/Amount/AmountCrypto.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ const AmountCrypto: React.FC<AmountProperties> = ({
 	}
 
 	return (
-		<span data-testid="AmountCrypto" className={cn("whitespace-nowrap", className)}>
+		<span data-testid="AmountCrypto" className={cn("space-x-1 whitespace-nowrap", className)}>
```
Comment (w3ea) [inline src/app/components/Amount/AmountCrypto.tsx:27]:
```suggestion
		<span data-testid="AmountCrypto" className={cn("whitespace-nowrap", className)}>
```
i think this is a leftover code
`formattedAmount` is string

File: src/domains/transaction/components/TransactionDetail/TransactionAmount/TransactionAmount.tsx (hunk 1 above)
Comment (w3ea) [reply src/domains/transaction/components/TransactionDetail/TransactionAmount/TransactionAmount.tsx]: yeah, i agree about this :ok_hand:

File: src/app/components/Amount/AmountCrypto.tsx (hunk 3 above)
Comment (dated) [reply src/app/components/Amount/AmountCrypto.tsx:27]: it is, yea. was testing some things :hide:
